.ci.yaml

```diff
@@ -3661,27 +3661,9 @@ targets:
 
   - name: Mac_x64 module_test_ios
     recipe: devicelab/devicelab_drone
-    bringup: true
-    timeout: 60
-    properties:
-      dependencies: >-
-        [
-          {"dependency": "ruby", "version": "ruby_3.1-pod_1.13"}
-        ]
-      tags: >
-        ["devicelab", "hostonly", "mac"]
-      task_name: module_test_ios
-    runIf:
-      - dev/**
-      - packages/flutter_tools/**
-      - bin/**
-      - .ci.yaml
-
-  - name: Mac_arm64 module_test_ios
-    recipe: devicelab/devicelab_drone
-    bringup: true
     timeout: 60
     properties:
+      cpu: x86 # Codesigning fails on ARM https://github.com/flutter/flutter/issues/112033
```

Comment:
Is this needed if the name is `Mac_x64`?

Reply:
I agree, also comment may be removed